.circleci/process_submitted_data.py

```diff
@@ -20,38 +20,39 @@
 
 
 for submitted_file in submitted_file_split:
-
-    if submitted_file.startswith('submitted_data'):
-        dir_name, data_type, file_name = submitted_file.split('/')
-        out_dir_name = 'processed_data'
-
-        if not os.path.isdir(out_dir_name):
-            os.makedirs(out_dir_name, exist_ok=True)
-        if not os.path.isdir(out_dir_name + '/' + data_type):
-            os.makedirs(out_dir_name + '/' + data_type, exist_ok=True)
-
-        outfile = submitted_file.replace(dir_name, out_dir_name)
-
-        if not os.path.isfile(outfile):
-            if not data_type == 'tad':
-                from cimr.processor.utils import Infiler
-                infile = Infiler(
-                    data_type, 
-                    submitted_file, 
-                    genome_build='b38', 
-                    update_rsid=False, 
-                    outfile=str(outfile),
-                    chunksize=10000000
-                )
-                infile.read_file()
-
-                if data_type == 'eqtl':
-                    from cimr.processor.query import Querier
-                    genes = list(infile.list_genes())
-                    queried = Querier(genes)
-                    queried.form_query()
-
-            else:
-                logging.info(f' processed file already exists for {submitted_file}')
-                logging.info(f' if reprocessing, delete {outfile} and file a new pull request')
-
+    # Ignore "submitted_data/downloaded_archive/" sub-dir
+    if submitted_file.startswith('submitted_data/downloaded_archive/'):
+        continue
+
```

Comment:
why is below showing up as new code?

Reply:
Removal of trailing spaces is one reason, another reason is due to the removal of this line:
```
if submitted_file.startswith('submitted_data'):
```
(because `submitted_file_split` is the result of walking through `submitted_data/` directory, so this `if` is always true). Thus the indentation on each line of the original `if` block is different.